.github/workflows/config.yml

```diff
@@ -128,7 +128,28 @@ jobs:
         run : |
           if [ $PACKS_CHANGED ]; then
             BRANCH_NAME=${REF#refs/heads/}
-            # TODO - Upload to the artifacts server of your choice.
+
+            # ========= UPLOAD TO ARTIFACTS SERVER OPTION =========
+
+            # TODO: Upload to the artifacts server of your choice.
+            # Create a file with the service account data
+            # use the bucket_upload script to upload your packs to google cloud storge
+            python $GITHUB_WORKSPACE/repository/build_related_scripts/bucket_upload.py --service_account $GITHUB_WORKSPACE/service_account.json --packs_directory $NEW_PACKS_FOLDER --branch_name $BRANCH_NAME
+            # Delete the service account file
+            rm $GITHUB_WORKSPACE/service_account.json
+
+            # ========= UPLOAD DIRECTLY TO YOUR XSOAR MACHINE (WHEN MERGING TO MAIN REPO) =========
+
+            if [ $BRANCH_NAME != master ]; then
+
+              CONFIG_FILE=$(cat xsoar_config.json)
+              MARKETPLACE_PACKS_LIST=$(cat $CONFIG_FILE | jq -r '.marketplace_packs')
+              # Upload Custom Packs
+              demisto-sdk upload --input-config-file /xsoar_config.json
+              # Upload MarketPlace Packs
+              python3 build_related_scripts/MarketPlaceInstallerFromCICD.py --marketplace-packs-list $MARKETPLACE_PACKS_LIST
+            fi
```

Comment:
Are they supposed to leave that part anyway? delete it if they choose the first option?
Needs further description